src/HaskellCI/GitHub.hs

```diff
@@ -480,6 +488,47 @@ makeGitHub _argv config@Config {..} prj jobs@JobVersions {..} = do
     withHaddock :: String
     withHaddock = "--with-haddock $HADDOCK"
 
+    ircJob :: [String] -> ListBuilder (String, GitHubJob) ()
+    ircJob serverChannelNames = item ("irc", GitHubJob
+        { ghjName            = "Haskell-CI (IRC notification)"
+        , ghjRunsOn          = "ubuntu-18.04"
+        , ghjNeeds           = [mainJobName]
+        , ghjIf              = Nothing
```

Comment:
This is where I can include `Just "${{ github.repository == 'ekmett/lens' }}"`?

Reply:
Yes. That being said, I should probably test this claim once it's implemented.